tox.ini

```diff
@@ -84,8 +89,24 @@ commands =
 
 [testenv:flake8]
 basepython = python2.7
-deps = flake8
-commands = flake8 --doctests src tests setup.py
+deps =
+    flake8
+    flake8-html
+    flake8-coding
+    flake8-debugger
+    flake8-deprecated
```

Comment:
This plug-in does not help here, as none if its [checks](https://github.com/gforcada/flake8-deprecated/blob/master/flake8_deprecated.py#L12) is relevant for RestrictedPython.